### PR TITLE
patch: Remove config package from client

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

File: client/lib/balena.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { BalenaSDK, DeviceTag } from 'balena-sdk';
+const { BalenaSDK, DeviceTag } = require('balena-sdk')
```
Comment (klutchell) [inline client/lib/balena.ts:1]: Why make these changes in this PR? They seem unrelated to the config package, at least make it a separate commit.

Comment (vipulgupta2048) [reply client/lib/balena.ts]: ready to review

File: client/jest.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
 	preset: 'ts-jest',
 	testEnvironment: 'node',
-};
+};
```
Comment (klutchell) [inline client/jest.config.js:4]: These files still appear as modified because the trailing newline was removed.

File: client/lib/balena.ts [hunk 3]
```diff
@@ -88,4 +88,4 @@ export class BalenaCloudInteractor {
 	async resolveDeviceUrl(device: DeviceInfo): Promise<string> {
 		return this.sdk.models.device.getDeviceUrl(device.deviceId);
 	}
-}
+}
```
Comment (klutchell) [inline client/lib/balena.ts:91]: Same here.

File: client/test/balena.test.ts [hunk 4]
```diff
@@ -41,4 +41,4 @@ describe('DeviceInfo', () => {
 	it('generates a file name prefix', () => {
 		expect(device.fileNamePrefix()).toStrictEqual('raspberrypi3-RPi0_wireless');
 	});
-});
+});
```
Comment (klutchell) [inline client/test/balena.test.ts:44]: and here...

File: client/tsconfig.json [hunk 5]
```diff
@@ -1,25 +1,26 @@
...
+    "include": [
+      "bin/**/*.*",
+      "lib/**/*.*",
+      "test/**/*.*",
+      "config.js"
+    ],
+    "exclude": [
+      "node_modules"
+    ]
+}
```
Comment (klutchell) [inline client/tsconfig.json:26]: By changing all the whitespace on this file I can't see what actually changed.

Comment (vipulgupta2048) [reply client/tsconfig.json:26]: Things got messed up locally. I had to revert and stash quite a bit of change because first I intended to do a PR with full client update (check branch name) then I thought to break it up into logical chunks and then I thought ... actually thought too much with a weekend coming in. Let me reset and open a new PR. Begin again.
